src/Components/ComponentUT.cpp

```diff
@@ -0,0 +1,75 @@
+﻿// MIT License
```

Comment:
Usually, the coding style specify tests files to be named as: something_test.cc
You can follow the coding style or have a consistent style for your project.